app/controllers/wpcc/your_contributions_controller.rb

```diff
@@ -3,17 +3,15 @@
 module Wpcc
   class YourContributionsController < EngineController
     protect_from_forgery
+    after_action :store_eligible_salary, only: :new
```

Comment:
But is it not an after action evident? I am don't want to make a big case out of this so I am happy to change though, even when I disagree with it.